posthog/temporal/batch_exports/temporary_file.py

```diff
@@ -662,6 +662,7 @@ def __init__(
         flush_callable: FlushCallable,
         schema: pa.Schema,
         compression: str | None = "snappy",
+        compression_level: int | None = None,
```

Comment:
do we make use of this setting at the moment or is this for use in future?

Reply:
I used it during testing, was curious if it had any impact on compressed size and/or cpu usage. In my testing, bumping it to the max (19) only produced a very small reduction of file size. So, for the moment, I decided to leave it unset, but maybe in the future we will want to tweak this.

Reply:
When unset, `pyarrow` decides which compression level to use, but the docs don't say how.

Reply:
Without any more data, I am just defaulting to whatever it decides.